### PR TITLE
Refactor the default keymap for crkbd and add a new keymap for me

### PR DESCRIPTION
**Pull request by foostan**
# Refactor the default keymap for crkbd and add a new keymap for me

### refactor the default keymap
- Remove `LAYOUT_kc` s from the default keymap(Use `LAYOUT`).
- Fix some keys(remove tap dance, add arrow keys).
- Remove self defined keys(e.g. `KC_CTLTB`, `KC_RST ` and more).
- Disable keylogs.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/foostan/keymap.c [hunk 1]
```diff
@@ -0,0 +1,242 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
```
Comment (noroadsleft) [inline keyboards/crkbd/keymaps/foostan/keymap.c]: The `QMK_KEYBOARD_H` include includes `bootloader.h` automatically.

```suggestion
```

Comment (foostan) [reply keyboards/crkbd/keymaps/foostan/keymap.c]: Thanks. fixed 🙏

**Comment by drashna** [comment]
Thanks! 


Well, while I have you here .... 

Any issues with this PR: https://github.com/qmk/qmk_firmware/pull/6284
It fixes a few issues that I found with the board. It's mostly until the "split common" PR is ready to merge (it's waiting on another PR).   But if you could approve that PR... :)

**Comment by foostan** [comment]
Thanks!

I will check it!

**Pull request merged**